static/app/views/issueDetails/streamline/metricIssueChart.tsx

```diff
@@ -0,0 +1,116 @@
+import {Fragment, lazy, useMemo} from 'react';
+import moment from 'moment-timezone';
+
+import {DateTime} from 'sentry/components/dateTime';
+import LazyLoad from 'sentry/components/lazyLoad';
+import {t} from 'sentry/locale';
+import type {Event} from 'sentry/types/event';
+import type {Group} from 'sentry/types/group';
+import type {Project} from 'sentry/types/project';
+import {formatMRIField} from 'sentry/utils/metrics/mri';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+import type {TimePeriodType} from 'sentry/views/alerts/rules/metric/details/constants';
+import {
+  getFilter,
+  getPeriodInterval,
+} from 'sentry/views/alerts/rules/metric/details/utils';
+import {
+  Dataset,
+  type MetricRule,
+  TimePeriod,
+} from 'sentry/views/alerts/rules/metric/types';
+import {extractEventTypeFilterFromRule} from 'sentry/views/alerts/rules/metric/utils/getEventTypeFilter';
+import {isCrashFreeAlert} from 'sentry/views/alerts/rules/metric/utils/isCrashFreeAlert';
+import {isCustomMetricAlert} from 'sentry/views/alerts/rules/metric/utils/isCustomMetricAlert';
+
+const MetricChart = lazy(
+  () => import('sentry/views/alerts/rules/metric/details/metricChart')
+);
+
+export function MetricIssueChart({
+  event,
+  group,
+  project,
+}: {
+  event: Event | undefined;
+  group: Group;
+  project: Project;
+}) {
+  const api = useApi();
+  const organization = useOrganization();
+
+  const ruleId = event?.contexts?.metric_alert?.alert_rule_id;
+  const {data: rule} = useApiQuery<MetricRule>(
+    [
+      `/organizations/${organization.slug}/alert-rules/${ruleId}/`,
+      {
+        query: {
+          expand: 'latestIncident',
+        },
+      },
+    ],
+    {
+      staleTime: Infinity,
+      retry: false,
+    }
+  );
+
+  const openPeriod = group.openPeriods?.[0];
+  const timePeriod = useMemo((): TimePeriodType | null => {
+    if (!openPeriod) {
+      return null;
+    }
+    const start = openPeriod.start;
+    let end = openPeriod.end;
+    if (!end) {
+      end = new Date().toISOString();
+    }
+    return {
+      start,
+      end,
+      period: TimePeriod.SEVEN_DAYS,
+      usingPeriod: false,
+      label: t('Custom time'),
+      display: (
+        <Fragment>
+          <DateTime date={moment.utc(start)} />
+          {' — '}
+          <DateTime date={moment.utc(end)} />
+        </Fragment>
+      ),
+      custom: true,
+    };
+  }, [openPeriod]);
+
+  if (!rule || !timePeriod) {
+    return null;
+  }
```

Comment:
DM'd about this, but since the rule is derived from an event, selecting a window without an occurrence (via chart zoom) can make the graph disappear (when the event query comes up empty).

This is an issue for crons/uptime as well, and is out of scope for now, but long term solution is to serialize these rule/detector IDs along with the group, rather than event, since that's how they're fingerprinted anyway.